scratchai/imgutils.py

```diff
@@ -243,8 +245,9 @@ def imshow(img, normd:bool=False, rz=224, **kwargs):
        Defaults to 224.
   """
   if isinstance(rz, int): rz = (rz, rz)
-  if isinstance(img, list):
+  if isinstance(img, list) or (isinstance(img, torch.Tensor) and len(img.shape) == 4):
```

Comment:
The first part allows for `list` of images to be processed and the second part allows a batch of images to be processed.